scripts/packDeps.sh

```diff
@@ -1,9 +1,9 @@
+#!/bin/bash -exv
+
 echo "Packing environment after success... this make take ~10mins or more"
-bash -c "while true; do echo -n .; sleep 5; done" &
 
 ORIG_PATH=`pwd`
 
-PROGRESS_REPORTER_PID=$!
```

Comment:
this PROGRESS_REPORTER is not needed. This was done to prevent timeout on travis CI which is not an issue anymore